app/models/custom_form_question.rb

```diff
@@ -1,4 +1,8 @@
 class CustomFormQuestion < ApplicationRecord
+  extend Mobility
+  translates :name
+  translates :option_list, type: :string # if this is json, ok to say string here?
```

Comment:
Wasn't sure how best to support translations of a nested hash/json/etc. Any suggestions?

Reply:
That sounds fine, but what does `option_list` hold again? It might need to be `:text` if the json could be on the longer side?

Reply:
If someone adds a select or multiselect input_type, they'll need to provide this option_list. i don't forsee that being super long, but it's definitely possible.

It isn't clear that the string_translations table could store any json, and that's what I was trying to get at. Wouldn't we have to change the string_translations migration for that @exbinary ? Or does it "just work" TM ?

Reply:
Another great question!

I don't think this will 'just work', not fully.

I'm guessing it'll pull the data out fine, since its string-like, but we won't be able to query against it with any type-specific operators. So eg, we wouldn't be able to use any array or json querying operators. Worth testing it out, probably.